### ISSUE TITLE
Heading H1 to H6 having no effect on text.

### ISSUE THREAD

**Issue by this-is-id** [jpuri/react-draft-wysiwyg#1240]
Hi, I've used this text editor in my Next js blog app, while everything else is working as intended. The heading from H1 to H6 are having no effect on text. 
Here is what it looks like when I apply heading. (Nothing happens)
![image](https://user-images.githubusercontent.com/95299143/156775942-640f0aab-0bab-41f6-b19c-767c77f3e402.png)

Here is the main code of text editor.
```
const onEditorStateChange = (editorState) => {
        setEditorState(editorState);
    };

<Editor 
            editorState={editorState}
            onEditorStateChange={onEditorStateChange}

            toolbarClassName='flex sticky top-0 z-50 !bg-[#e6f4f885] 
            !justify-center mx-auto '
            editorClassName="bg-[#ffffffe5] mt-6 shadow-lg max-w-5xl mx-auto mb-12 border p-10"
            />
```
Can anybody guide why is heading not working?

**Comment by bamichoi** [comment]
I have the same issue now.

**Comment by codedog001** [comment]
In my case this issue was due to tailwind CSS. If you're facing similar issue due to tailwind consider integrating slate js rich text editor instead of this one.

**Comment by bamichoi** [comment]
@codedog001 Thank you, exactly I use tailwindCSS. But is it a unique solution integrating slate js or other editors at this moment?

**Comment by codedog001** [comment]
I'm not sure whether switching to slate js is the only solution or not, but it worked for me. Actually what you'll need to do is add classname="text-2xl" or maybe text-xl, basically the tailwind style classname in h1, h2 ... h6, to make it work. Now, as far as I know, I found no such place in this editor to configure this, while in slate js editor, this was configurable. If you switch to slate js, let me know, I'll share the code on how to make headings work in it. But, keep in mind, in slate js, I am still not able to make unordered and ordered lists work, they are still TODO.

**Comment by santhosh-umapathi-appymango** [comment]
Good News Tailwind Users,

I have found a workaround for this issue,

Override the ```globals.css``` file where you integrate all the tailwind base components.

Paste the below and you are good to go:
```bash
@layer base {
  h1 {
    @apply text-4xl;
    @apply font-bold;
  }
  h2 {
    @apply text-3xl;
    @apply font-bold;
  }
  h3 {
    @apply text-2xl;
    @apply font-bold;
  }
  h4 {
    @apply text-xl;
    @apply font-semibold;
  }
  h5 {
    @apply text-md;
    @apply font-semibold;
  }
  h6 {
    @apply text-sm;
    @apply font-semibold;
  }
}
```

**Comment by Mikhail1Demianenko** [comment]
> Good News Tailwind Users,
> 
> I have found a workaround for this issue,
> 
> Override the `globals.css` file where you integrate all the tailwind base components.
> 
> Paste the below and you are good to go:
> 
> ```shell
> @layer base {
>   h1 {
>     @apply text-4xl;
>     @apply font-bold;
>   }
>   h2 {
>     @apply text-3xl;
>     @apply font-bold;
>   }
>   h3 {
>     @apply text-2xl;
>     @apply font-bold;
>   }
>   h4 {
>     @apply text-xl;
>     @apply font-semibold;
>   }
>   h5 {
>     @apply text-md;
>     @apply font-semibold;
>   }
>   h6 {
>     @apply text-sm;
>     @apply font-semibold;
>   }
> }
> ```
> 
> Reference: https://tailwindcss.com/docs/adding-custom-styles#adding-base-styles

Hey!
Im facing the same issue here
I was thinking about the same solution you posted.
It does not seem right to me to change the whole design system for the sake of one component
I've been trying to find a way to override base styles for one component only, but I found  nothing

**Comment by bamichoi** [comment]
@santhosh-umapathi-appymango Thank you, It works!  I had to change "text-md " to "text-lg". the tailwindcss complained it does not exist in the base style that it offers. But it works well! Fortunately, I don't have a custom design for text at this moment. Thank you again!

**Comment by santhosh-umapathi-appymango** [comment]
> @santhosh-umapathi-appymango Thank you, It works! I had to change "text-md " to "text-lg". the tailwindcss complained it does not exist in the base style that it offers. But like @Mikhail1Demianenko said, it is apply whole heading tags. so, using this solution, we have to change other h tags we used if we don't want to apply this style.

Yeah, i totally forgot, i added a custom font size by extending the config on the tailwind configs: ```tailwind.config.js```
 ```bash
 fontSize: {
        md: ["18px", "24px"],
        "8.5xl": ["7.5rem", "1"],
        "10xl": ["9rem", "1"],
      }
```

**Comment by santhosh-umapathi-appymango** [comment]
> > Good News Tailwind Users,
> > I have found a workaround for this issue,
> > Override the `globals.css` file where you integrate all the tailwind base components.
> > Paste the below and you are good to go:
> > ```shell
> > @layer base {
> >   h1 {
> >     @apply text-4xl;
> >     @apply font-bold;
> >   }
> >   h2 {
> >     @apply text-3xl;
> >     @apply font-bold;
> >   }
> >   h3 {
> >     @apply text-2xl;
> >     @apply font-bold;
> >   }
> >   h4 {
> >     @apply text-xl;
> >     @apply font-semibold;
> >   }
> >   h5 {
> >     @apply text-md;
> >     @apply font-semibold;
> >   }
> >   h6 {
> >     @apply text-sm;
> >     @apply font-semibold;
> >   }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Reference: https://tailwindcss.com/docs/adding-custom-styles#adding-base-styles
> 
> Hey! Im facing the same issue here I was thinking about the same solution you posted. It does not seem right to me to change the whole design system for the sake of one component I've been trying to find a way to override base styles for one component only, but I found nothing

Agreed ! this solution can be applied when you are free to customize the Tailwind Design System on h tags. In my case, it was not much of breaking change on my project.

**Comment by afrilabs-capacity** [comment]
Add the following to your stylesheet.
Worked for me.

.DraftEditor-editorContainer .public-DraftEditor-content h1{
font-size:  2em!important;
font-weight: bolder!important;
}

.DraftEditor-editorContainer .public-DraftEditor-content h2{
    font-weight: bolder!important;
font-size: 1.5em!important;
}

.DraftEditor-editorContainer .public-DraftEditor-content h3{
font-weight: bolder!important;	
font-size: 1.7em!important;
    
}

.DraftEditor-editorContainer .public-DraftEditor-content h4{
font-size: 1em!important;
font-weight: bolder!important;
}

.DraftEditor-editorContainer .public-DraftEditor-content h5{    	
font-size: .83em!important;
font-weight: bolder!important;
}

.DraftEditor-editorContainer .public-DraftEditor-content h6{    	
font-size: .67em!important;
font-weight: bolder!important;
}

**Comment by Irfanwani** [comment]
I realised that any other ui library messes with the html tags,

**Comment by vinitGithub099** [comment]
>Even this is not working in my case.

**Comment by Amandeep4567** [comment]
<h1 className=" font-semibold text-5xl"> Hello User</h1> simply we can use this, It works.

**Comment by Amandeep4567** [comment]
`<h1 className="font-semibold text-5xl"> Hello User </h1>`
simply we can use this, It works.

**Comment by Karthi981** [comment]
[https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes](https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes) Here you go just apply those h1,h2 styles for particular components.

**Comment by Nightvision53** [comment]
> Add the following to your stylesheet. Worked for me.
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h1{ font-size: 2em!important; font-weight: bolder!important; }
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h2{ font-weight: bolder!important; font-size: 1.5em!important; }
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h3{ font-weight: bolder!important; font-size: 1.7em!important;
> 
> }
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h4{ font-size: 1em!important; font-weight: bolder!important; }
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h5{ font-size: .83em!important; font-weight: bolder!important; }
> 
> .DraftEditor-editorContainer .public-DraftEditor-content h6{ font-size: .67em!important; font-weight: bolder!important; }

This worked like charm for me. Thanks!

**Comment by mariful-nyx** [comment]
```
h1, h2, h3, h4, h5, h6 {
  font-size: revert;
  font-weight: revert;
}
```

**Comment by ConradoAlmeida** [comment]
> ```
> h1, h2, h3, h4, h5, h6 {
>   font-size: revert;
>   font-weight: revert;
> }
> ```

This one works to me!
Thanks!

**Comment by bilalbutt044** [comment]
> ```
> h1, h2, h3, h4, h5, h6 {
>   font-size: revert;
>   font-weight: revert;
> }
> ```

this is will revert the style for whole project. i would suggest to add a class of element so that just the children of that element style  revert
